solutions/logging/client-experimentation/setters.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+######
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  client-name: 'client1' # lowercase only
+  client-displayname: 'Client1'
+  client-folderviewer: 'group:client1@example.com'
+  #############
+  # logging Project ID
+  logging-prj-id: logging-prj-id-12345
+  #############
+  # LoggingLogBucket retention settings
+  # Set the number of days to retain logs in Cloud Logging buckets
+  # Set the lock mechanism on the bucket to: true or false
+  # After a retention policy is locked (true), you can't delete the bucket until every log in the bucket has fulfilled the bucket's retention period
+  # AU-9 PROTECTION OF AUDIT INFORMATION
+  # AU-11 AUDIT RECORD RETENTION
+  retention-locking-policy: false
```

Comment:
do we know the required retention values to meet guardrails?
if so, should we pre-set them as defaults?

Reply:
If you wanted to test out an LZ deploy or just this package for example and set these values retention-locking-policy: true and retention-in-days: 365 you could end up locking your testing environment for 365 days. As soon as logs would be written into the logs buckets, you would need to wait for 365 days for the data to expire before being able to delete the logging project.